Index: node_modules/sparse-bitfield/.npmignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/node_modules/sparse-bitfield/.npmignore b/node_modules/sparse-bitfield/.npmignore
new file mode 100644
--- /dev/null	(date 1704736110159)
+++ b/node_modules/sparse-bitfield/.npmignore	(date 1704736110159)
@@ -0,0 +1,1 @@
+node_modules
